tools/Makefile.win

```diff
@@ -178,6 +178,8 @@ BIN       = $(NUTTXNAME)$(EXEEXT)
 
 all: $(BIN)
 .PHONY: dirlinks context clean_context check_context configenv config oldconfig menuconfig nconfig export subdir_clean clean subdir_distclean distclean apps_clean apps_distclean
+.PHONY: pass1 pass1deps pass1dep
+.PHONY: pass2 pass2deps pass2dep
```

Comment:
let's merge into one line since line 180 is alrady very long.

Reply:
Sorry for jumping in but isn't that hardly readable 
I propose that we split in on several lines